src/neighbor.cpp

```diff
@@ -1408,7 +1408,7 @@ void Neighbor::init_topology()
 void Neighbor::print_pairwise_info()
 {
   int i,m;
-  char str[128];
+  char str[256];
```

Comment:
missing the changes of `sprintf` to `snprintf` in this file.

Reply:
Increasing the buffer seemed sufficient in this case.

Reply:
It's not clear to me why this function even uses sprintf, unless there are requests for which [none of these branches](https://github.com/lammps/lammps/blob/ca04e8f31c62224dc8b01d19d229d21bf0cbe89a/src/neighbor.cpp#L1458-L1470) are entered.  It seems it could just use `fprintf`.

Reply:
@ExpHP **very** good point. I'll make that change.

Reply:
@rbberger this is now fixed properly